docs/get_started/get_started_demos.md

```diff
@@ -1,48 +1,45 @@
 # Basic OpenVINO™ Workflow {#openvino_docs_get_started_get_started_demos}
```

Comment:
```suggestion
# Basic OpenVINO™ Workflow (C++) {#openvino_docs_get_started_get_started_demos}
```

Reply:
I agree that "Get Started with C++ Samples" is a better title for this page. However, we will also have to change it in the sidebar defined by the `toctree` section in [Samples_Overview.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/Samples_Overview.md) and anywhere else it's mentioned documentation. Is that something you can do?

Reply:
@EdjeElectronics Nice catch! I'm afraid that I'm not able to add a new file to this PR. I can only do that via a new PR. Can you help make that change?

Reply:
Okay, I changed it in the `toctree` on that page. I noticed that you've already changed it in other places in documentation (like the new "What's Next?" sections on the install pages), nice!